test/jdk/java/awt/Scrollbar/AquaLFScrollbarTest/ScrollBarBorderTest.java

```diff
@@ -0,0 +1,196 @@
+import javax.swing.JScrollBar;
+import javax.swing.JPanel;
+import javax.swing.JFrame;
+import javax.swing.BoxLayout;
+import javax.swing.JLabel;
+import javax.swing.Box;
+import javax.swing.UIManager;
+import javax.swing.border.Border;
+import javax.swing.border.EmptyBorder;
+import java.awt.Frame;
+import java.awt.GridBagLayout;
+import java.awt.GridBagConstraints;
+import java.awt.Panel;
+import java.awt.TextArea;
+import java.awt.Button;
+import java.awt.Scrollbar;
+import java.awt.Component;
+import java.awt.Graphics;
+import java.awt.Color;
+import java.awt.Insets;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+/*
+ * @test
+ * @run main/manual ScrollBarBorderTest
```

Comment:
Please add short summary of the test with "@summary" tag and BugID with "@bug". You can see how it is done in any other OpenJDK test.

Reply:
Probably it will be good to try some kind of automation.